hikvision_tftpd.py

```diff
@@ -34,20 +35,20 @@ class Server(object):
     _TFTP_OPCODE_ACK = 4
     _TFTP_OPCODE_OACK = 6
     _TFTP_ACK_PREFIX = struct.pack('>h', _TFTP_OPCODE_ACK)
-    BLOCK_SIZE = 512
 
     def __init__(self, handshake_addr, tftp_addr, filename, file_contents):
         self._file_contents = file_contents
         self._filename = filename
-        self._total_blocks = ((len(file_contents) + self.BLOCK_SIZE)
-                              // self.BLOCK_SIZE)
+        self._block_size = _DEFAULT_BLOCK_SIZE
+        self._total_blocks = ((len(file_contents) + self._block_size)
+                              // self._block_size)
         self._tftp_rrq_prefix = (struct.pack('>h', self._TFTP_OPCODE_RRQ) +
                                  filename + '\x00')
-        self._tftp_blksize_option = ('blksize' + '\x00')
+        self._tftp_blksize_option = ('blksize\x00')
```

Comment:
the parenthesis aren't necessary either. (even on the statement before, the outer parenthesis are only needed because it spans lines.)